application/src/main/kotlin/net/corda/v5/application/flows/FlowContextProperties.kt

```diff
@@ -0,0 +1,45 @@
+package net.corda.v5.application.flows
+
+/**
+ * Context properties of a [Flow] are key value pairs of Strings. They are comprised of two sets of properties, those
+ * set by the platform when a [Flow] is started, and those which are added by the CorDapp developer during the execution
+ * of their flow. The latter set are referred to as user context properties. All context properties are immutable once
+ * set.
+ * Where keys exist as platform properties already, setting user properties is an effective no-o as they will always be
```

Comment:
"setting user properties is an effective no-o as they will always be", there is probably a better way to write this for public apis 😛

Reply:
typo, should have been no-op, but I assume you also mean that is inappropriate for the public api so I will change it